app/assets/stylesheets/v3/layout_components/pre-footer.scss

```diff
@@ -0,0 +1,102 @@
+.pre-footer {
+  @include grid-column(12,0);
+  background: $white;
+  height: $side-nav-height;
+  top: $top-bar-height;
+  z-index: 9000;
+
+  @include breakpoint(medium down) {
+    display: none;
+  }
+
+  .logo {
+    background-image: asset-url(ama-logo);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin-left: 2*$base-margin;
+    object-fit: contain;
+    padding: $base-padding;
+    width: 100px;
+  }
+
+  &__spacer {
+    @include large-2;
+    height: 100%;
+    position: absolute;
+    z-index: -1; //to prevent overlap with notifications
+
+    &--white {
+      @extend .side-nav__spacer;
+      background-color: $white;
+    }
+
+    &--show-for-medium {
+      @extend .side-nav__spacer;
+      @include breakpoint(medium down) {
+        display: block;
+      }
+    }
+  }
+
+  &__header {
+    @extend .heading;
+    background: $cerulean;
+    color: $white;
+  }
+
+  &__list {
+    list-style-type: none;
+    margin: 0;
+    cursor: default;
+  }
+
+  &__content {
+    background: $bg-grey-light;
+    padding: $base-padding*2 $base-padding*2;
```

Comment:
I noticed the content seems more indented than the main body content, can we maybe change the left and right padding to be just 1 rem?